0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3472 @@
+{
```

Comment:
<p>Could the initialisation be made worse? (This lets the user hone in more on what a good fit is). Could the increments be made smaller for the slider? (e.g. see code). Also could the plot be a bit more zoomed in so that the trend is more clear? </p><pre class="ql-syntax" spellcheck="false">b = -2.0 #@param {type:"slider", min:-5, max:5, step:0.5}
w = -0.02 #@param {type:"slider", min:-0.05, max:0.05, step:0.005}
print("Plotting line", w, "* x +", b)
parameters = [b, w]
parameters_list.append(parameters)
plot_basic_data(parameters_list,
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; title="Observed data and my first predictions",axis_pad=10)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='135'/>

Reply:
<p>Thanks! I have updated the init. I made the axis_pad=12 (for some values of w, when using axis_pad=10, it goes off the plot). </p>